crates/symbolicator/src/cli.rs

```diff
@@ -87,10 +87,11 @@ pub fn execute() -> Result<()> {
         release,
         session_mode: sentry::SessionMode::Request,
         auto_session_tracking: false,
-        traces_sampler: Some(Arc::new(|ctx| {
+        traces_sampler: Some(Arc::new(move |ctx| {
             if Some(true) == ctx.sampled() {
                 1.0
             } else if ctx.operation() != "http.server" {
+                // TODO: Think about what to do with this comment now:
```

Comment:
I think you can move this comment to the definition of the config option, or to the `Default` impl.

Reply:
Moved it to the `Default` impl.